packages/self-service/embed-example.html

```diff
@@ -19,9 +19,9 @@
         j = d.createElement(s);
         j.defer = true;
         j.type = 'module';
-        j.src = 'https://aichatbot.sendbird.com/index.js';
+        j.src = './dist/index.js';
         f.parentNode.insertBefore(j, f);
-      }(window, document, 'script', 'A3E33A85-E4A7-46A4-99B1-EF6833E7DE3A', 'marketing_bot_3');
```

Comment:
Will it be better if I just keep this? Otherwise we might need to look for  appid / botid everytime from somewhere else 😅

Or I can use the Khan academy one in the .env.

Reply:
For testing purposes, it looks good to keep this!
Additionally, to prevent issues in recent deployments, it would be good to have apps and files per plan (starter/pro). For example, embed-example-pro.html and embed-example-starter.html.

Reply:
Yeah sounds good. I'll prepare them too in a separate PR :)